packages/plans-grid/src/plans-table/index.tsx

```diff
@@ -44,7 +44,9 @@ const PlansTable: React.FunctionComponent< Props > = ( {
 	customTagLines,
 	defaultAllPlansExpanded = false,
 } ) => {
-	const supportedPlans = useSelect( ( select ) => select( PLANS_STORE ).getSupportedPlans() );
+	const supportedPlans = useSelect( ( select ) =>
+		select( PLANS_STORE ).getPeriodSupportedPlans( 'ANNUALLY' )
+	);
```

Comment:
We should do this change also in the accordion version because right now Gutenboarding (`/new/plans-modal`) is showing 8 paid plans.

Reply:
☝️the same apply also for `PlansDetails`.